editor.planx.uk/src/@planx/components/MapAndLabel/Public.tsx

```diff
@@ -1,17 +1,62 @@
 import { alpha } from "@mui/material/styles";
+import Typography from "@mui/material/Typography";
+import { Feature } from "geojson";
 import { useStore } from "pages/FlowEditor/lib/store";
-import React from "react";
+import React, { useEffect, useState } from "react";
+import { FONT_WEIGHT_SEMI_BOLD } from "theme";
 
 import Card from "../shared/Preview/Card";
 import CardHeader from "../shared/Preview/CardHeader";
-import { MapContainer } from "../shared/Preview/MapContainer";
+import { MapContainer, MapFooter } from "../shared/Preview/MapContainer";
 import { PublicProps } from "../ui";
 import { MapAndLabel } from "./model";
 
 type Props = PublicProps<MapAndLabel>;
 
+export type Boundary = Feature | undefined;
+
 function MapAndLabelComponent(props: Props) {
   const teamSettings = useStore.getState().teamSettings;
+  const [boundary, setBoundary] = useState<Boundary>();
+  const [area, setArea] = useState<number | undefined>(0);
+  const [objectArray, setObjectArray] = useState<any>([]);
```

Comment:
I'm not sure I understand the purpose of `objectArray` here (variable name is quite vague & it's never rendered)? Is boundary sufficient alone?

Reply:
It's a way for me to track the event listener dispatching events and collecting info into an array. Most likely negated in a PR or two when multiple items get added to the map and we can use the geoJSON features, but helpful for now if I wanted to test adding a list of items to the Public comp... is it too rough to add in the PR? @jessicamcinchak

Reply:
Thanks for explanation - happy for it to merge here now better understanding it's work-in-progress :slightly_smiling_face: